nx-workspace-emotion/libs/tailwind-config/src/lib/theme.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-undef */
+module.exports = {
+  theme: {
+    extend: {
+      colors: {
+        zabadoo: 'blue',
```

Comment:
Here's where we're defining a special variable, in this case, a brand colour called "zabadoo".